ia/src/protocole.py

```
@@ -118,8 +99,17 @@ def handle_client(client) -> None:
                 if (message == "dead\n"):
                     print("Client dead, closing connection")
                     handle_Dead(client, message)
-                if (message and client["commandes"]):
-                    handle_commande(client, client["commandes"][0], message)
+
+                if (client["commandes"]):
```

Pylint (3.3.7)

[convention] Unnecessary parens after 'if' keyword

(C0325)